src/els_dt_documents.erl

```diff
@@ -0,0 +1,83 @@
+%%==============================================================================
+%% The 'documents' table
+%%==============================================================================
+
+-module(els_dt_documents).
+
```

Comment:
Isn't the `-behaviour(els_db_table).` missing from all `*_dt_*` modules?

Reply:
🤦‍♂